src/dot_utils.rs

```diff
@@ -82,15 +82,15 @@ fn attr_map_to_string<'a>(
     if attrs.is_empty() {
         return Ok("".to_string());
     }
-
     let attr_string = attrs
         .iter()
         .map(|(key, value)| {
-            if key == "label" {
-                format!("{}=\"{}\"", key, value)
-            } else {
-                format!("{}={}", key, value)
-            }
+            let escaped_value = value
+                .replace('\\', "\\\\") // Escape backslashes first
```

Comment:
Is there a dictionary of values that we should escape? Just wondering

Reply:
There is a library we can make use serde_json library to do this instead

Reply:
Ideally there is 6
https://static.rust-lang.org/doc/master/reference.html#byte-escapes

Reply:
> Is there a dictionary of values that we should escape? Just wondering

Ideally there are 6, 
https://static.rust-lang.org/doc/master/reference.html#byte-escapes